app/build.gradle

```diff
@@ -15,7 +15,7 @@ apply plugin: 'kotlin-android'
 apply plugin: 'kotlin-android-extensions'
 apply plugin: 'kotlin-kapt'
 apply plugin: 'io.fabric'
```

Comment:
- As Fabric is deprecated, use Firebase Crashlytics instead of Fabric Crashlytics
- Also make sure as this is new project, use all the latest version of dependencies

Reply:
- updated fabric Crashlytics to Firebase Crashlytics
- updated all dependencies